app/utils/validatePhoneNumber.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ *  @module normalize/validatePhoneNumber
+ * Validator for US only phone numbers.
+ */
+
+// eslint-disable-next-line import/no-unused-modules
+const PHONE_REGEX = /^\(?(\d{3})\)?[- ]?(\d{3})[- ]?(\d{4})$/;
+
+module.exports = {
```

Comment:
Please use ES6-style exports (i.e. the `export` keyword) instead of CommonJS exports (i.e. `module.exports`). Our app is not a Node.js application, so we don't need compatibility with that ecosystem, and even if we ever did need to, because we run our app through Webpack anyway, we could always ask Webpack to output a Node.js-compatible bundle, even if we use ES6 exports internally.